Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members  = ["./crates/*", "tasks/*"]
+members  = ["./crates/*", "crates/rolldown_workspace", "tasks/*"]
```

Comment:
!

Reply:
NIce catch, it is auto-generated by `cargo new`